src/Makefile

```diff
@@ -156,7 +156,7 @@ endif
 ifeq (${FC},ifort)
     # replace -xHost with -march=core-avx2 to make sure the same code builds on
     # both Intel and AMD CPUs 
-    OPTS         += -g -O2 -march=core-avx2 -ip -assume byterecl -fp-model precise -ftz -init=snan,arrays -traceback -no-fma
+    OPTS         += -g -O2 -xHost -ip -assume byterecl -fp-model precise -ftz -init=snan,arrays -traceback -no-fma
```

Comment:
Could we revert this back to the -march=core-avx2 option?  The current setting will not work for AMD processors.

Reply:
Arg yes, I keep forgetting this before pushing